pages/api/load.ts

```diff
@@ -1,23 +1,61 @@
-import { NextApiRequest, NextApiResponse } from 'next';
-import { encodePayload, getBCVerify, setSession } from '../../lib/auth';
+import { NextApiRequest, NextApiResponse } from "next";
+import { checkAppExtensionsExist } from "@lib/appExtensions";
+import { getStoreToken } from "@lib/dbs/firebase";
+import { encodePayload, getBCVerify, setSession } from "../../lib/auth";
 
 const buildRedirectUrl = (url: string, encodedContext: string) => {
-    const [path, query = ''] = url.split('?');
-    const queryParams = new URLSearchParams(`context=${encodedContext}&${query}`);
+  const [path, query = ""] = url.split("?");
```

Comment:
I think we prefer single quotes unless the string contains a single quote. Looks like all your quotes are double. Could be auto-linting in your IDE. This might actually be something we should add to eslint config.